advanced_filters/forms.py

```diff
@@ -305,6 +305,10 @@ def __init__(self, *args, **kwargs):
         self.initialize_form(instance, self._model, data, extra_form)
 
     def clean(self):
+        for form in self.fields_formset:
+            if not form.is_valid():
+                form.cleaned_data = {'field': 'id', 'DELETE': True}
```

Comment:
I struggle to see why should the `cleaned_data` of all forms that fail to validate be equal to the above dictionary?

Could you kindly create a test reproducing the issue and/or add additional context?